"Get text inside textfield ()" build fail

### Issue description

```
> Task :compileJava FAILED 
C:\Users\sucki\MCreatorWorkspaces\fabrictestenv\src\main\java\net\mcreator\fabrictestenv\network\HandheldWikiGuiButtonMessage.java:36: error: cannot find symbol 
                  .execute(com.google.common.collect.ImmutableMap.<String, Object>builder().put("guistate", guistate).build()); 
                                                                                                            ^ 
  symbol:   variable guistate 
  location: class HandheldWikiGuiButtonMessage 
1 error 
FAILURE: Build failed with an exception.
```

![image](https://user-images.githubusercontent.com/17712352/205703073-fcd9673d-7ece-4637-8582-fa1e41be85c6.png)

### How to reproduce this issue?

Make GUI element - add text field - add procedure that uses said text field - test.

### Operating system

Windows

### Details

Currently using fabric generator 1.19.2, version 1.7, on newest release of mcreator 2022.3.

### Example workspace

_No response_

### Logs

_No response_

### Issue tracker rule checks (please read carefully)

- [X] I have checked that my problem <a href='https://github.com/MCreator/MCreator/issues?q=is%3Aissue' target='_blank'>is not already reported</a>
- [X] I have checked that my problem is not covered on <a href='https://mcreator.net/support/knowledgebase' target='_blank'>Knowledge Base</a> or on <a href='https://mcreator.net/wiki' target='_blank'>MCreator's Wiki</a>
- [X] I have checked that my written content does not violate the <a href='https://mcreator.net/wiki/general-publishing-guidelines' target='_blank'>publishing guidelines</a>

--- a/src/fabric-1.19.2/templates/gui/gui_msg_button.java.ftl
+++ b/src/fabric-1.19.2/templates/gui/gui_msg_button.java.ftl
@@ -51,6 +51,7 @@ public class ${name}ButtonMessage extends FriendlyByteBuf {
 		double z = buf.readInt();
 		server.execute(() -> {
 		Level world = entity.getLevel();
+		HashMap guistate = ${name}Menu.guistate;
 		<#assign btid = 0>
 		<#list data.components as component>
 			<#if component.getClass().getSimpleName() == "Button">

--- a/src/fabric-1.19.2/templates/gui/gui_msg_slot.java.ftl
+++ b/src/fabric-1.19.2/templates/gui/gui_msg_slot.java.ftl
@@ -55,6 +55,7 @@ public class ${name}SlotMessage extends FriendlyByteBuf {
 		int meta = buf.readInt();
 		server.execute(() -> {
 			Level world = entity.getLevel();
+			HashMap guistate = ${name}Menu.guistate;
 			<#list data.components as component>
 				<#if component.getClass().getSimpleName()?ends_with("Slot")>
 					<#if hasProcedure(component.onSlotChanged)>

